software/triliumnext-notes.yml

```
@@ -0,0 +1,12 @@
+name: TriliumNext Notes
+website_url: https://github.com/TriliumNext/Notes
```

Let's use here the Doc Website
```suggestion
website_url: https://triliumnext.github.io/Docs/
```

---

> [!IMPORTANT] The docs repo is archived while we transition to a new documentation mechanism.

As soon as they get the docs updated, I'll update the entry.

---

Ups, didn't see this.